client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCache.java

```diff
@@ -3,6 +3,7 @@
 import org.infinispan.commons.api.BasicCache;
 import org.infinispan.commons.util.CloseableIterator;
 import org.infinispan.commons.util.concurrent.NotifyingFuture;
+import org.infinispan.query.dsl.Query;
```

Comment:
Same case of ```ClientEvents``` that relies heavily on query/protostream but has a non-query method ```.mkCachefailoverEvent()```: unless you don't call the query methods, the loader should work